rest_api/openshift_v1.adoc

```diff
@@ -9278,7 +9278,7 @@ Patch is provided to give a concrete name and type to the Kubernetes PATCH reque
 |certificate|provides certificate contents|false|string|
 |key|provides key file contents|false|string|
 |caCertificate|provides the cert authority certificate contents|false|string|
-|destinationCACertificate|provides the contents of the ca certificate of the final destination.  When using re-encrypt termination this file should be provided in order to have routers use it for health checks on the secure connection|false|string|
+|destinationCACertificate|provides the contents of the ca certificate of the final destination.  When using re-encrypt termination this file should be provided in order to have routers use it for health checks on the secure connection|true|string|
```

Comment:
Thanks, @pweil-.

@ahardin-rh @jkaurredhat To me then it seems then that it should stay `false` in the API Guide, and just make sure the "Re-encryption Termination" section in the Arch topic makes it clear that `destinationCACertificate` is required, but only for re-encryption (which this PR is on its way towards doing).